tern/formats/generator.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+import abc
+
+
+class GenerateReport(object):
+    '''Base class for report plugins'''
+    def __init__(self):
+        '''Do nothing'''
```

Comment:
Line: 16
    pylint: pointless-string-statement / String statement has no effect (col 8)

Do you need an `__init__`?

Reply:
Removed __init__.